.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+ATF Test Scripts to check #[issue number]
+
+This PR is **[ready / not ready]** for review.
+
+### Summary
+[Summary of PR changes]
+
+### ATF version
+[ATF version to use]
+
+### Changelog
+##### Breaking Changes
```

Comment:
in my opinion "Breaking Changes" is not applicable for scripts, so can be removed.

Reply:
@GetmanetsIrina Removed in d1e0828